src/main/java/net/mcreator/ui/views/editor/image/versioning/change/Change.java

```diff
@@ -49,7 +49,8 @@ public UUID getGroup() {
 		return group;
 	}
 
-	public void setUUID(UUID group) {
+	public Change setUUID(UUID group) {

```

Comment:
Is this change really necessary?

Reply:
This way one can get rid of 2 extra lines:
https://github.com/MCreator/MCreator/blob/894d50c06073128b27ff3743615ca50d62286751/src/main/java/net/mcreator/ui/dialogs/imageeditor/ResizeDialog.java#L63